core/ui/src/main/java/com/msg/ui/StudentActivityCard.kt

```diff
@@ -0,0 +1,103 @@
+package com.msg.ui
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.msg.design_system.theme.BitgoeulAndroidTheme
+import com.msg.model.remote.enumdatatype.ApproveStatus
+import com.msg.model.remote.enumdatatype.Authority
+import com.msg.model.remote.model.activity.InquiryStudentActivityModel
+import com.msg.ui.util.toKoreanFormat
+import java.time.LocalDate
+import java.util.UUID
+
+@Composable
+fun StudentActivityCard(
+    data: InquiryStudentActivityModel,
+    onClick: (UUID) -> Unit,
+    role: Authority = Authority.ROLE_USER
+) {
+    BitgoeulAndroidTheme { colors, typography ->
+        Card(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(154.dp)
+                .clickable(
+                    onClick = { onClick(data.activityId) }
+                ),
+            colors = CardDefaults.cardColors(
+                containerColor = colors.WHITE
+            ),
+            border = BorderStroke(1.dp, colors.G9)
+        ) {
+            Box(modifier = Modifier.padding(16.dp)) {
+                Column(
+                    modifier = Modifier
+                        .fillMaxSize()
+                )
+                {
```

Comment:
```suggestion
                ) {
```
이거 혹시 이게 더 lint에 맞지 않을까요?